### ISSUE TITLE
`New major version is available: 2.16+183b29a`

### ISSUE THREAD

**Issue by jackfischer** [edgedb/edgedb-cli#1069]
- EdgeDB Version: `2.16+183b29a`
- EdgeDB CLI Version: `3.1.1+5798f31`
- OS Version: macos 13.4

Steps to Reproduce:

1. Project on 2.16
2. run `edgedb project upgrade`
3. claims `2.16+183b29a` is the new major version, recommends doing a `--to-latest`, which would presumably take me to either the 3.x RC or 4.x dev, both of which would be unexpected since they're not stable.

```
$ cat edgedb.toml
[edgedb]
server-version = "2.16"
$ edgedb project upgrade
EdgeDB instance is up to date with the specification in the `edgedb.toml`.
New major version is available: 2.16+183b29a
To update `edgedb.toml` and upgrade to this version, run:
    edgedb project upgrade --to-latest
```

**Comment by tailhook** [comment]
`--to-latest` does not update to the testing channel. But you're right that the messaging is weird. Fixed in #1072

**Comment by jackfischer** [comment]
My mistake, I was thinking of `--to-nightly`
